### ISSUE TITLE
remark-lint-no-tabs broken - id.split is not a function

### ISSUE THREAD

**Issue by Zamiell** [remarkjs/remark-lint#278]
### Initial checklist

- [X] I read the [support docs](https://github.com/remarkjs/.github/blob/main/support.md)
- [X] I read the [contributing guide](https://github.com/remarkjs/.github/blob/main/contributing.md)
- [X] I agree to follow the [code of conduct](https://github.com/remarkjs/.github/blob/main/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Aremarkjs&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

remark-lint-no-tabs 3.1.1

### Link to runnable example

n/a

### Steps to reproduce

- `git clone git@github.com:hanabi/hanabi.github.io.git`
- `cd hanabi.github.io.git`
- `git checkout upgrade`
- `npm ci`
- `npx remark docs`

Observe that it says "no issues found" for every file.

Next, open the `.remarkrc.mjs` file in the root of the repository and uncomment line 27:
```
// "remark-lint-no-tabs", // TODO BROKEN
```

Next, run `npx remark docs`, and observe that it gives the following error for every file:

```
misc\todo.md
  1:1  error  Error: Cannot parse file `.remarkrc.mjs`
Cannot import `node_modules\remark-lint-no-tabs\index.js`
TypeError: id.split is not a function
    at lintRule (file:///D:/Repositories/hanabi.github.io/node_modules/remark-lint-no-tabs/node_modules/unified-lint-rule/lib/index.js:25:20)
    at file:///D:/Repositories/hanabi.github.io/node_modules/remark-lint-no-tabs/index.js:101:26
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async loadFromAbsolutePath (file:///D:/Repositories/hanabi.github.io/node_modules/unified-engine/lib/configuration.js:429:20)
    at async addModule (file:///D:/Repositories/hanabi.github.io/node_modules/unified-engine/lib/configuration.js:349:20)
    at async use (file:///D:/Repositories/hanabi.github.io/node_modules/unified-engine/lib/configuration.js:319:7)
    at async addEach (file:///D:/Repositories/hanabi.github.io/node_modules/unified-engine/lib/configuration.js:288:7)
    at async addPreset (file:///D:/Repositories/hanabi.github.io/node_modules/unified-engine/lib/configuration.js:263:7)
    at FormattedError (file:///D:/Repositories/hanabi.github.io/node_modules/fault/index.js:35:12)
    at file:///D:/Repositories/hanabi.github.io/node_modules/unified-engine/lib/find-up.js:190:15
    at done (file:///D:/Repositories/hanabi.github.io/node_modules/trough/index.js:145:7)
```

Also, the issue form that you have is broken, because it only has npm 7 instead of npm 8.

### Expected behavior

It should not have a runtime error.

### Actual behavior

It has a runtime error.

### Runtime

Node v16

### Package manager

Other (please specify in steps to reproduce)

### OS

Windows

**Comment by wooorm** [comment]
run `npm update`

**Comment by Zamiell** [comment]
@wooorm Every dependency in my package.json is already up to date, and yet my issue still remains. Can you be more specific and/or reopen this issue?

**Comment by wooorm** [comment]
As mentioned, `npm update`. You updated `remark-lint-no-tabs`, which wants `unified-lint-rule` 2+. But you have 1.
